stable/nginx-ingress/templates/nginx-service.yaml

```diff
@@ -0,0 +1,26 @@
+{{- $root := . -}}
+{{- range .Values.nginx.services }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+{{ toYaml .annotations | indent 4}}
+  labels:
+    app: "{{ template "fullname" $root }}"
+    chart: "{{ $root.Chart.Name }}-{{ $root.Chart.Version }}"
+    component: controller
+    heritage: {{ $root.Release.Service }}
+    release: {{ $root.Release.Name }}
+  name: "{{ template "fullname" $root }}-controller-{{ .name }}"
+spec:
```

Comment:
This needs an optional externalIPs section. This would allow it to easily be able to be bound to vips for clusters that use those.